code/WorkInProgress/TempEngine.dm

```diff
@@ -156,6 +172,7 @@
 		//Calculate necessary moles to transfer using PV = nRT
 		var/pressure_delta = (input_starting_pressure - output_starting_pressure)/2
 
+		src.warning_active = 0
```

Comment:
```suggestion
		src.warning_active = FALSE
```

Reply:
MIXED FEELINGS! 😱

Reply:
u put it as FALSE before!

Reply:
Agreed.
I think that was a holdover from initial prototype with a binary indication. Setting a bit field to false felt odd, but the variable naming and some usage is more self documenting this way.

I lean more towards the reviewer in general to produce an artifact that is understandable by... not me.  👍